### DISCUSSION TITLE
GitHub Education

### DISCUSSION THREAD

**Discussion by ao-shuo** [community/community#138448] [GitHub Education]
### Hi everyone,

As the question states, I've applied for over a week and I haven't received my benefits yet, nor any confirmation email. I'm sure my application process is correct as shown in the attached picture. After uploading my student certification, I received "Approved" on the web page, but I still cannot use benefits. can you tell me what to do

**Comment by masterwww1** [comment]
It sounds frustrating to be in this situation. Here are a few steps you can take:

Double-Check Your Submission: Ensure that your application was submitted correctly. Verify that all required documents were uploaded and that there are no errors or missing information.
Check Spam/Junk Email: Sometimes, confirmation emails might end up in your spam or junk folder. Make sure to check these folders in your email.
Contact Customer Support: Reach out to the support team of the organization handling the benefits. Provide them with your application details and explain the issue. They should be able to assist you in resolving the problem.
Verify Application Status: If the organization has an online portal or app where you can check the status of your application, use it to see if there are any updates or additional steps required.
Wait for Processing Time: Sometimes, there can be a delay in processing. Check if there is an estimated processing time provided by the organization and wait a bit longer if needed.

If you need any specific contact details or assistance with drafting a follow-up message, let me know!

**Comment by ao-shuo** [comment]
I have been successfully approved for education certification, and the materials have been submitted completely, but I have not received any confirmation email. I can confirm that there is no problem with my material submission, and I now need the student discount provided by GitHub. Could you please help me solve it?
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "community/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月12日(星期四) 中午11:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [community/community] GitHub Education (Discussion #138448)





 
It sounds frustrating to be in this situation. Here are a few steps you can take:
 
Double-Check Your Submission: Ensure that your application was submitted correctly. Verify that all required documents were uploaded and that there are no errors or missing information.
 Check Spam/Junk Email: Sometimes, confirmation emails might end up in your spam or junk folder. Make sure to check these folders in your email.
 Contact Customer Support: Reach out to the support team of the organization handling the benefits. Provide them with your application details and explain the issue. They should be able to assist you in resolving the problem.
 Verify Application Status: If the organization has an online portal or app where you can check the status of your application, use it to see if there are any updates or additional steps required.
 Wait for Processing Time: Sometimes, there can be a delay in processing. Check if there is an estimated processing time provided by the organization and wait a bit longer if needed.
 
If you need any specific contact details or assistance with drafting a follow-up message, let me know!

**Comment by davevad93** [comment]
Hi @ao-shuo ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ao-shuo was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
